Шестая лабораторная работа выполнена

PopovMatvey opened the pull request:
Были выполнены:
1) все задания с первого по шестой;
2) все тесты и и линтеры.

PopovMatvey committed:
Шестая лабораторная работа выполнена

Были выполнены:
1) все задания с первого по шестой;
2) все тесты и и линтеры.

djilua1 commented on src/ex6_js-array-methods/task-01.js:
```diff
@@ -0,0 +1,13 @@
+function sliceAnalog(array, begin, end) {
+  if (end === undefined) {
+    return array.slice(begin);
```
Так нельзя. :)
Смысл полифилла в том, что метод не успользуем,

djilua1 commented on src/ex6_js-array-methods/task-06.js:
```diff
@@ -0,0 +1,9 @@
+  return array.reduce(callback, initialValue);
+}
+
+module.exports = reduceAnalog;
```
Нужно все переделать, т.к. ты используешь нативные методы, а нужно написать их реализацию.

PopovMatvey committed:
Исправления по шестой лабораторной работе

1) Была исправлена реализация всех методов для которых было поставлено задание (с первого по шестое);
2) Тесты и линтеры также все пройдены.
